### PR TITLE
Update GUI Layout; move JSON converter to menu bar

### LINKED ISSUE #93: Move Frc-Char JSON converter to Menu Bar
This will leave more space for the other widgets.

### PR DESCRIPTION
**Pull request by Piphi5**
Updates GUI to make room for the generator and moves FRC-Char JSON to a popup that can be activated from the menu bar.
Fixes #93 
Overall Layout:
image

Converter Location:
image

Popup:
image

### CODE REVIEW

File: src/main/native/cpp/Main.cpp [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
+#include <imgui.h>
```
Comment (calcmogul) [inline src/main/native/cpp/Main.cpp:5]: Why was this added? Should it be moved down with the other includes?

Comment (Piphi5) [reply src/main/native/cpp/Main.cpp:5]: I'm not too sure as to why it was added. I think VSCode might've done it on its own?

Comment (calcmogul) [reply src/main/native/cpp/Main.cpp:5]: Since we're using imgui calls directly, we should probably keep it but sort it with the other headers.

**Review by calcmogul — APPROVED** [review]
